WelcomeHome/WelcomeHome.DAL/Models/EventType.cs

```diff
@@ -1,11 +1,16 @@
 ﻿using System.ComponentModel.DataAnnotations;
 
-namespace WelcomeHomeModels.Models
+namespace WelcomeHome.DAL.Models
 {
     public class EventType
     {
         [Key]
         public int Id { get; set; }
         public string Name { get; set; } = null!;
+        public List<Event> Events { get; set; }
+        public EventType()
+        {
+            Events = new List<Event>();
```

Comment:
I think we can just make this field nullable, but then initialize when it is needed